app/src/main/java/com/findmyrecycling/ProfileScreen.kt

```diff
@@ -38,22 +43,37 @@ fun ProfileOptions(name: String) {
             value = profile,
             onValueChange = {profile = it},
             label = { Text(stringResource(R.string.profile)) },
-            modifier = Modifier.fillMaxWidth()
-        )
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(start = 4.dp, end = 4.dp, top = 2.dp, bottom = 2.dp)
+                .border(width = 1.dp,
+                    color = Color.Black,
+                    shape = RoundedCornerShape(4.dp)),
 
+        )
 
         OutlinedTextField(
             value = mySavedLocations,
             onValueChange = { mySavedLocations = it },
             label = { Text(stringResource(R.string.mySavedLocations)) },
-            modifier = Modifier.fillMaxWidth()
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(start = 4.dp, end = 4.dp, top = 2.dp, bottom = 2.dp)
+                .border(width = 1.dp,
+                    color = Color.Black,
+                    shape = RoundedCornerShape(4.dp)),
```

Comment:
Used trailing commas as recommended in Kotlin coding conventions here:  https://kotlinlang.org/docs/coding-conventions.html#trailing-commas